plotly/tests/test_core/test_graph_objs/test_frames.py

```diff
@@ -56,20 +58,37 @@ def test_deeply_nested_layout_attributes(self):
         )
 
     def test_deeply_nested_data_attributes(self):
-        frames = Frames()
-        frames.append({})
-        frames[0].data = [Bar()]
-        frames[0].data[0].marker.color = 'red'
+        #frames = Frames()
+        #frames.append({})
+        #frames[0].data = [Bar()]
+        #frames[0].data[0].marker.color = 'red'
+
+        frames = Frame
+        frames.data = [Bar()]
+        frames.data[0].marker.color = 'red'
```

Comment:
This doesn't seem right to me. The new syntax is replacing this with a list of `Frame` objects,
```
help(go.Frames)
        plotly.graph_objs.Frames is deprecated.
Please replace it with a list or tuple of instances of the following types
  - plotly.graph_objs.Frame
```
So, this test should be:
```
frames = []
frames.append(go.Frame())
frames[0].data = [go.Bar()]
frames[0].data[0].marker.color = 'red'
```

Reply:
Although `go.Frame()` doesn't seem to work:
```
In [11]: go.Frame()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-11-d2b9d2e3e8b5> in <module>()
----> 1 go.Frame()

~/Repos/plotlypy-jmease/plotly.py/plotly/graph_objs/_frame.py in __init__(self, baseframe, data, group, layout, name, traces, **kwargs)
    181         # ---------------------
    182         self._validators['baseframe'] = v_frame.BaseframeValidator()
--> 183         self._validators['data'] = v_frame.DataValidator()
    184         self._validators['group'] = v_frame.GroupValidator()
    185         self._validators['layout'] = v_frame.LayoutValidator()

~/Repos/plotlypy-jmease/plotly.py/plotly/validators/frame/_data.py in __init__(self, plotly_name, parent_name)
      6     def __init__(self, plotly_name='data', parent_name='frame'):
      7         super().__init__(
----> 8             plotly_name=plotly_name, parent_name=parent_name, role='object'
      9         )

TypeError: __init__() got an unexpected keyword argument 'role'
```
Am I doing something wrong here? cc @jmmease

Reply:
Nope, I just broke it :-) Fixed in 8a7b476dded4d56d94549c4dae2d007095fcb833